lib/card.rb

```diff
@@ -0,0 +1,12 @@
+class Card
+	attr_reader :question,
+							:answer,
+							:category
+	def initialize (question, answer, category)
+		@question = question
+		@answer = answer.downcase
+		@category = category
+
+	end
+
+end
```

Comment:
This looks good. I like that you're downcasing your answer for edge cases. Make sure to check your spacing before pushing up to github, as syntax is a bit off.